prime-router/metadata/fhir_mapping/hl7/datatype/XONToOrganization.yml

```diff
@@ -0,0 +1,124 @@
+resourceType: Organization
+
+id:
+  type: STRING
+  valueOf: "GeneralUtils.generateResourceId()"
+  expressionType: JEXL
+
+name:
+  type: STRING
+  valueOf: XON.1
+  expressionType: HL7Spec
+
+extension_1-nameType:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: organization-name-type
+    valueCodeableConcept:
+      valueOf: datatype/CodeableConcept
+      generateList: true
+      expressionType: resource
+      specs: XON.2
+
+identifier_1:
```

Comment:
This is great 👏 One nit: the order of sub-expressions seems a little random, could we get those in order by HL7 field index to make this a little easier to take inventory of?

Reply:
Will update